Assets/Source/Entities/Behaviors/Influential.cs

```diff
@@ -1,14 +1,33 @@
 using UnityEngine;
 using System.Collections;
+using System.Collections.Generic;
 
 public class Influential : MonoBehaviour
 {
+	private static HashSet<Influential> queens = new HashSet<Influential>();
+	public static HashSet<Influential> Queens {
+		get {
+			return queens;
+		}
+	}
+
 	/* A single "strength" modifier. This encompasses both the power and distance of transmission.
 	* For now, merely specifies distance, but later the algorithm should be tweaked appropriately.
 	*/
 	public float strength = 100;
 
+	private Renderer renderer;
 	private GameObject influenceBubble = null;
+	private Player player;
+
+	public Player Player {
```

Comment:
Property shorthand does the same thing: http://stackoverflow.com/questions/687340/c-sharp-shorthand-property-question

```csharp
public Player Player { get; set; }
```